### ISSUE TITLE
Erro em página temática

### ISSUE THREAD

**Issue by mformiga** [servicosgovbr/portal-de-servicos#529]
1. Como ponto focal crio página temática
2. Como admin não consigo visualizar a página recentemente criada
3. Limpo todos os dados de navegação 
4. Entro no editor novamente como admin e consigo visualizar a página temática.

**Comment by mformiga** [comment]
Também não é possível visualizar a página temática  recém criada no IE 11.

**Comment by gustavo-freitas** [comment]
@mformiga por favor, detalhar mais os cenários descritos. Esse cenário contempla dois usuários com perfis diferentes utilizando o Editor de Serviços ao mesmo? Além disso, quais dados de navegação você se refere? Quando não é possível visualizar página temática no IE 11?

**Comment by mformiga** [comment]
@gustavo-freitas Sim. Dois usuários com perfis distintos (ponto focal e admin). O primeiro cria a página. O segundo vai tentar publicar mas não consegue visualizar pois não aparece na lista. Limpo todo o cache do navegador, histórico de acesso, senhas, cookies, etc

**Comment by gustavo-freitas** [comment]
@mformiga analisamos o cenário descrito e descobrimos os seguintes pontos:

1. Existem duas instâncias para o Editor de Serviços na atual infra-estrutura do Portal de Serviços
2. Toda vez que um usuário acessa o Editor de Serviços, o balanceador de carga direciona o usuário para a instância 1 ou instância 2 do Editor de Serviços, de acordo com disponibilidade de cada instância
3. Quando um usuário salva um serviço, estando em uma determinada instância, esse serviço é salvo em uma branch local e uma branch remota
4. Caso outro usuário estiver utilizando o Editor de Serviços, em outra instância, esse serviço salvo pelo usuário anterior não será exibido. Pois, somente os branches locais são exibidas. Essa decisão técnica foi tomada levando em consideração a performance da aplicação
5. Entretanto, quando o mesmo usuário logar novamente no Editor de Serviços, os branches que não eram exibidos anteriormente, agora serão listados e poderão sofrer edições normalmente.

Tendo em vista que, o cenário descrito não introduz problemas nos dados da aplicação, não altera a experiência do usuário de maneira drástica e possui uma solução de contorno (logar novamente no Editor), não consideramos esse cenário como algo crítico. 

De qualquer forma, uma solução para esse cenário foi detalhada e implementada parcialmente. Maiores detalhes podem ser encontrados no seguinte pull request: https://github.com/servicosgovbr/editor-de-servicos/pull/512

**Comment by josselmo** [comment]
Funcionou.
Homologado.
